fe/fe-core/src/main/java/com/starrocks/leader/ReportHandler.java

```diff
@@ -890,6 +888,11 @@ private static void deleteFromMeta(ListMultimap<Long, Long> tabletDeleteFromMeta
             }
         } // end for dbs
 
+        if (deleteTablets.size() > 0) {
+            GlobalStateMgr.getCurrentState().getEditLog()
+                    .logBatchDeleteReplica(new BatchDeleteReplicaInfo(backendId, deleteTablets));
```

Comment:
need to put it in lock?

Reply:
There is no need to put it in db lock. If the related meta is dropped, the replay code can cover this case.